lib/ansible/modules/cloud/amazon/iam_password_policy.py

```diff
@@ -61,7 +61,8 @@
     aliases: [allow_password_change]
   pw_max_age:
     description:
-      - Maximum age for a password in days.
+      - Maximum age for a password in days.  To remove the requirement set to 0
```

Comment:
The user shouldn't need to know that the module is 'removing' the requirement, just what the end result is. Perhaps something like `Maximum age for a password in days. When this option is 0 the password has no expiration.`

Reply:
"When this option is 0 then passwords do not expire automatically."